pages/new-pedal.js

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import NewPedalForm from "../components/new-pedal-form";
 
-export default function AddPedal() {
+export default function AddPedal({ pedals, handlePedalSubmit }) {
   return (
     <>
-      <h1>New Pedal \o/</h1>
-      <NewPedalForm />
+      <header>
+        <h1>New Pedal 🤩</h1>
+      </header>
+      <NewPedalForm pedals={pedals} handlePedalSubmit={handlePedalSubmit} />
```

Comment:
ah look. you did this already. 
For better namingconvention, this could be 
```suggestion
      <NewPedalForm pedals={pedals} onHandlePedalSubmit={handlePedalSubmit} />
```